src/aura/STG_CMP_Addr/STG_CMP_AddrHelper.js

```diff
@@ -26,5 +26,16 @@
 		    }
 		});
 		$A.enqueueAction(runBatchAction);
-	},
+    },
+    handlePhoneSync : function (component, event) {
+        var enablePrefPhone = event.getSource().get("v.value");
+        
+        if(enablePrefPhone) {
+        	component.set("v.prefPhoneErrorsDisabled", false);
```

Comment:
This can be component.set("v.prefPhoneErrorsDisabled", !enablePrefPhone); ?

Reply:
I'm not sure. If enablePrefPhone doesn't have a value, is the var null or is it false? I'm terrible at javascript but I like that we're setting false here only if enablePrefPhone evaluates to true.

Reply:
It's also easier for me to read it this way...

Reply:
I was going for readability -- thought I kind of do the way @xinnanlisf was suggesting when prototyping the bugfix I was working on. However it doesn't take into account a possible "null" and I love being explicit.

Reply:
Actually, isn't it supposed to be better to do the double-equals in JS? Or even triple, if you know you have the correct data type...

Reply:
I think triple is probably the best practice.

Reply:
Not sure if we need the helper, could we just use this  disabled="{!not(v.hierarchySettings.Enable_New_Preferred_Phone_Sync__c)}" . It disables the check box if the other check box is not enabled.